scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/psi/PSI.scala

```diff
@@ -14,45 +14,30 @@
  * limitations under the License.
  */
 
-package com.intel.analytics.bigdl.ppml.vfl
+package com.intel.analytics.bigdl.ppml.psi
```

Comment:
change to `com.intel.analytics.bigdl.ppml.algorithms.PSI`

similarly, `com.intel.analytics.bigdl.ppml.algorithms.vfl.LogisticRegression`, `com.intel.analytics.bigdl.ppml.algorithms.vfl.LinearRegression`, etc.